### ISSUE TITLE
Make sure all command line tools understand .gz files

### ISSUE THREAD

**Issue by BridgeAR** [MaibornWolff/codecharta#2580]
# Bug

## Expected Behavior

GIVEN a folder of cc.json.gz files

WHEN trying to merge them (and probably more commands)

THEN all files are combined

## Actual Behavior

The .gz files are not possible to be read by the merge filter.

## Steps to Reproduce the Problem

Use `ccsh merge`

## Specifications
- is released in [online-demo](https://maibornwolff.github.io/codecharta/visualization/app/index.html?file=codecharta.cc.json&file=codecharta_analysis.cc.json): yes 
- CodeCharta Version: 1.85.0
- OS: all
- Browser: not applicable

**Comment by phanlezz** [comment]
Hi @BridgeAR I have question on the general CLI experience (not directly related to this issue). I state the current situation, the extracted file name, and what behavior I would expect: 

1.  User gives `-o testABC` => Result: `testABC.gz` (file name: `testABC`) Expected: `testABC.cc.json.gz` (file name `testABC.cc.json`)
2. User gives `-o test.gz` => Result: `test.gz.gz` (file name: `test.gz`) Expected: maybe `test.gz`?

So basically: should we always append the file extension `cc.json` automatically, or should the given name be taken as is? And if the file extension is given completely by the user, i.e. `.cc.json.gz`, should we take it over as well, or make sure not to have a double `.gz`?

**Comment by BridgeAR** [comment]
Fixed by #2793
